src/main/java/net/testudobank/MvcController.java

```diff
@@ -405,7 +412,7 @@ public String submitWithdraw(@ModelAttribute("user") User user) {
     int userOverdraftBalanceInPennies = TestudoBankRepository.getCustomerOverdraftBalanceInPennies(jdbcTemplate, userID);
     if (userWithdrawAmtInPennies > userBalanceInPennies) { // if withdraw amount exceeds main balance, withdraw into overdraft with interest fee
       int excessWithdrawAmtInPennies = userWithdrawAmtInPennies - userBalanceInPennies;
-      int newOverdraftIncreaseAmtAfterInterestInPennies = (int)(excessWithdrawAmtInPennies * INTEREST_RATE);
+      int newOverdraftIncreaseAmtAfterInterestInPennies = (int)applyInterestRateToPennyAmount(excessWithdrawAmtInPennies);;
```

Comment:
You can just apply the `(int)` cast in the helper method itself, and make the helper method return an `int` instead of a `double` to make this even cleaner.

Reply:
Thank you, that's a good point.